spec/acceptance/mod_passenger_spec.rb

```diff
@@ -9,19 +9,8 @@
 
     case fact('operatingsystem')
     when 'Ubuntu'
-      case fact('lsbdistrelease')
-      when '14.04'
-        passenger_root = '/usr/lib/ruby/vendor_ruby/phusion_passenger/locations.ini'
-        passenger_default_ruby = '/usr/bin/ruby'
-      when '16.04'
-        passenger_root = '/usr/lib/ruby/vendor_ruby/phusion_passenger/locations.ini'
-        passenger_default_ruby = '/usr/bin/ruby'
-      else
-        # Includes 10.04 and 12.04
-        # This may or may not work on Ubuntu releases other than the above
-        passenger_root = '/usr'
-        passenger_ruby = '/usr/bin/ruby'
-      end
+      passenger_root = '/usr/lib/ruby/vendor_ruby/phusion_passenger/locations.ini'
+      passenger_default_ruby = '/usr/bin/ruby'
     when 'Debian'
```

Comment:
Since Wheezy is not mentioned in metadata.json you could simplify just static variables.

Reply:
Removed the version case statement, just static variables now